.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1038 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I've been out of the .NET world for a while, but this file seems obscenely large. Is it really necessary?